### DISCUSSION TITLE
Remove Triple Click from AP disconnect sound

### DISCUSSION THREAD

**Discussion by DescendDescend** [flybywiresim/aircraft#3153] [Ideas]
Triple click is for Mode Reversions, Inconsistencies in modes e.g. in DES and you select an altitude higher than current (as one example) and landing capability downgrade eg: Cat 3 to Cat 1. Long list in FCOM.

**Comment by hotshotp** [comment]
Yep, we know. I have added separate wwise events, its just not really a priority right now
